.github/workflows/ci.yaml

```diff
@@ -38,6 +43,31 @@ jobs:
       run: |
         meson test -t 5 --print-errorlogs -C ${{ matrix.os }}
 
+    - name: Build Debian source package
+      run: |
+        set -ex
+        mkdir deb
+        # keep version number as 0 for upstream, to match debian/changelog
+        git archive -o deb/sscg_0.orig.tar.gz --prefix sscg/ HEAD
+        cd deb
+        # unpack tar again to make sure the dist is complete and clean
+        tar xf sscg_*.tar.gz
+        cd sscg
+        cp -r packaging/debian .
+
+        # build source and binary package, with lintian
+        debuild -us -uc
```

Comment:
This doesn't honor the CI's intent to run with `${{ matrix.compiler }}`. It gets built on gcc, regardless of the setting.

Reply:
Good point, thanks -- I'll add the missing `export CC`. Will send a fix tomorrow.

Reply:
Current version does that properly now, the [package build uses clang](https://github.com/sgallagher/sscg/pull/22/checks?check_run_id=2290739581#step:7:49) (where it should)